src/main/java/codeu/controller/FileUploadServlet.java

```diff
@@ -0,0 +1,41 @@
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+
+public class FileUploadServlet extends HttpServlet {
+
+	@Override
+	public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+
+		//get the file chosen by the user
+		Part filePart = request.getPart("fileToUpload");
```

Comment:
On my pull request I added onto your profile.jsp code. In the parameter, instead of "fileToUpload",  "photo" should work to get the image.

Reply:
This is change now, thanks!